src/popup/fetch.ts

```diff
@@ -26,6 +26,21 @@ function roundToNextHour(date: Date): Date {
 	return date;
 }
 
+function reformatDate(date_str: string): string {
+	/*
+		Problem:  Notion does not convert times into the correct timezone,
+				  even when supplied a timezone, Notion will show the user the
+				  UTC time (incorrect time, unless you live in UTC region), labeled
+				  with the supplied timezone.
+		Solution: Re-format the ISO date string to give Notion a 'false' date string based
+				  on the offset of the local machine. This is a misuse of ISO time/date strings,
+				  but it will result in the correct date being shown in Notion.
+	 */
+	const date = new Date(date_str);
+	date.setMinutes(date.getMinutes() - date.getTimezoneOffset());
```

Comment:
Looks good, and tested as working—just wondering, would it be possible to tie this to the configured `Timezone` option?

Reply:
This can be done. It would take a decent amount of time to get the correct offset for the timezone manually entered, mainly because I would have to somehow figure out if that manually entered timezone is currently observing Daylight Savings, but if I can figure that out, I can get the correct offset.

Reply:
Would you be able to use `moment-tz` and parse the UTC date in the configured `Timezone` to get the offset date, or something similar?

The additional build size isn't really a concern for a browser extension, as it's loaded from disk rather than served over the web.

If not (or if it's too much hassle), I'm happy with this---will just require a footnote in the documentation.

Cheers!

Reply:
Done! Now the provided function will get the offset based on the user's provided timezone instead of the timezone of their machine. Then that offset is used to calculate an accurate date string to hand off to Notion.

Reply:
P.S. Thanks for the `moment-tz` suggestion, that's exactly what I was looking for!